src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write code here
+const names = document.querySelectorAll('li');
+
+function sortList(personList) {
+  return personList.sort((a, b) => b.salary - a.salary);
+}
+
+function getEmployees(people) {
+  return [...people].map(person => {
+    return {
+      name: person.textContent.replace(/\n/g, '').trim(),
+      position: person.dataset.position,
+      salary: +person.dataset.salary.slice(1).split(',').join(''),
+      age: +person.dataset.age,
+    };
+  });
+}
+
+sortList(getEmployees(names));
```

Comment:
fix parameter naming. You pass here `names`, and takes `people` in function. Looks like you you pass array of strings and takes array of object